content/python/concepts/math-module/terms/math-gcd/math-gcd.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'math.gcd()' 
+Description: 'Calculates the Greatest Common Divisor (GCD) of two integers.' 
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Discrete Math'
+Tags:
+  - 'Functions'
+  - 'Algorithms'
+  - 'Integers'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
```

Comment:
```suggestion
---
Title: 'math.gcd()' 
Description: 'Returns the Greatest Common Divisor (GCD) of two integers.' 
Subjects: 
  - 'Computer Science'
  - 'Data Science'
  - 'Discrete Math'
Tags:
  - 'Algorithms'
  - 'Functions'
  - 'Integers'
CatalogContent: 
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

Reply:
Changed the description with a word due to plagiarism issues and arranged the tags in an alphabetically ascending order